raiden-ts/src/channels/epics.ts

```diff
@@ -727,14 +729,12 @@ export const channelOpenEpic = (
                 // errors on makeDeposit$ are handled independently and don't fail open request
                 // but the channelDeposit.failure action may be fired
                 makeDeposit$(
-                  tokenContract,
-                  tokenNetworkContract,
-                  onchainAddress,
-                  address,
-                  partner,
+                  [tokenContract, tokenNetworkContract],
```

Comment:
I basically don't car too much. But because I'm curious I ask: what is your advantage of grouping these parameters into arrays? :thinking:

Reply:
Just to reduce the number of parameters on the function, and aggregate them when it makes sense. It actually just look like an Array, but works like a tuple, first of the 2 contracts, and then of the 3 addresses.

Reply:
If you like it. :shrug: